### PR TITLE
Fix extract hackage info

### LINKED ISSUE #883: Popularity estimation of Hackage packages does not work anymore
It looks like Hackage maintainers have changed [this page][hackage-browse] and it is now populated with JavaScript. This makes the instructions in `extract-hackage-info/README.md` obsolete. We need a way to make this work again.

Perhaps a script to automate the whole process of extraction would make sense, too.

[hackage-browse]: https://hackage.haskell.org/packages/browse

### PR DESCRIPTION
**Pull request by mrkkrp**
Close #883.

### CODE REVIEW

File: extract-hackage-info.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+OUTPUT="$WDIR/hackage-info.json"
+
+trap cleanup 0 1 2 3 15
+
+cleanup()
+{
+    rm -rf "$WDIR"; exit
+}
+
+nix-build -A extractHackageInfo --no-out-link
```
Comment (amesgen) [inline extract-hackage-info.sh:17]: I think this line should be unnecessary, as `extractHackageInfo` is built as part of `EXTRACTION_APP`?

**Review by amesgen — APPROVED** [review]
Looks good to me, there is probably no better way without a better Hackage API.

**Review by tbagrel1 — APPROVED** [review]
Looks good to me too!

File: extract-hackage-info.sh (hunk 1 above)
Comment (amesgen) [reply extract-hackage-info.sh:17]: > I wasn't sure that only the final build output goes to stdout.

Yes, the logs go to stderr; we already use this e.g. here:

https://github.com/tweag/ormolu/blob/276dfdd9a22dfad914578984b891d2a6d33300e6/ormolu-live/deploy.sh#L3

Comment (mrkkrp) [reply extract-hackage-info.sh]: Fixed :+1:

**Pull request merged**